src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildListener.java

```diff
@@ -61,7 +67,19 @@ public void onCompleted(AbstractBuild<?, ?> abstractBuild, @Nonnull TaskListener
     } else {
       buildUrl = rootUrl + abstractBuild.getUrl();
     }
-    repository.deletePullRequestComment(cause.getPullRequestId(), cause.getBuildStartCommentId());
+
+    // Delete the "Build Started" comment, as it gets replaced with a comment
+    // about the build result. Failure to delete the comment is not fatal, it's
+    // reported to the build log.
```

Comment:
This comment is OK, though I think we could do with an even shorter one. Something like `handle here, because it's not fatal` inside the catch block

Reply:
I'll keep it in mind for the future. Let's just leave the comments as is.